Tests/MapboxDirectionsTests/DirectionsTests.swift

```diff
@@ -113,16 +117,19 @@ class DirectionsTests: XCTestCase {
         
         let directions = Directions(credentials: BogusCredentials)
         let opts = RouteOptions(locations: [one, two])
-        directions.calculate(opts, completionHandler: { (response, error) in
+        directions.calculate(opts, completionHandler: { (session, result) in
             expectation.fulfill()
```

Comment:
fulfilling expectation before other checks seems incorrect...

Reply:
Why do you think that? Keep in mind that the fulfillment is not there to say "okay test is over", It's merely there to ensure that the (faked) directions request takes no longer than two seconds to call it's `completionHandler.`

Reply:
I was all ready to explain why you're wrong, but as it turns out, you're not. 

Very good catch. 🙇‍♂️